tests/test_table.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+import docx
+
+from html2docx import html2docx
+
+from .utils import TEST_DIR
+
+
+def test_table():
+    html_path = os.path.join(TEST_DIR, "table.html")
```

Comment:
Can `test_html2docx` be extended to accept tables in the spec?

The specs would need to be updated to specify paragraphs **and tables** in a document.
Since table cells can contain other tables or multiple paragraphs, we could recursively validate paragraphs and tables in a container (for now, a document or table cell).